src/nnet3/nnet-simple-component.h

```diff
@@ -120,17 +120,21 @@ class DropoutComponent : public RandomComponent {
                         Component *to_update,
                         CuMatrixBase<BaseFloat> *in_deriv) const;
   virtual Component* Copy() const { return new DropoutComponent(dim_,
-                                                                dropout_proportion_); }
+                                                                dropout_proportion_,
+                                                                dropout_per_frame_); }
   virtual std::string Info() const;
 
-  void SetDropoutProportion(BaseFloat dropout_proportion) { dropout_proportion_ = dropout_proportion; }
+  void SetDropoutProportion(BaseFloat dropout_proportion, bool dropout_per_frame) {
+     dropout_proportion_ = dropout_proportion;
+     dropout_per_frame_ = dropout_per_frame;
+      }
```

Comment:
Odd indentation.  Make sure you are not introducing tabs into the file.
If you use emacs, use
```
(setq-default tab-width 4)
(setq-default fill-column 80)
(setq-default indent-tabs-mode `nil)
(add-hook 'write-file-hooks 'delete-trailing-whitespace)


(load-file "~/.google-c-style.el")
(add-hook 'c-mode-common-hook 'google-set-c-style)
(add-hook 'c-mode-common-hook 'google-make-newline-indent)
```
You can find `google-c-style.el` from a web search.